frontend/public/components/secrets/create-secret.tsx

```diff
@@ -367,6 +698,43 @@ export type BasicAuthSubformProps = {
   },
 };
 
+export type ImageSecretFormState = {
+  type: SecretType,
+  stringData: {
+    [key: string]: any,
```

Comment:
Shouldn't this be (here and below)

```ts
[key: string]: string,
```

Probably worth making `stringData` a separate type you can reuse

Reply:
for the `ImageSecretFormProps` it makes sense to have it `{[key: string]: string}`, not sure whats the right type for `ImageSecretFormState`, since the `stringData` looks like: 
```json
{  
   ".dockerconfigjson":{  
      "auths":{  
         "https://index.docker.io/v1/":{  
            "username":"testuser",
            "password":"testpw",
            "auth":"dGVzdHVzZXI6dGVzdHB3",
            "email":"jhadvig@test.com"
         },
         "https://index.docker.io/v2/":{  
            "username":"testuser",
            "password":"testpw",
            "auth":"dGVzdHVzZXI6dGVzdHB3",
            "email":"jhadvig2@test.com"
         }
      }
   }
}
```

tried `{[key: string]: Object}` but that was throwing type errors:
```
TS2326: Types of property 'stringData' are incompatible.
  Type 'Object' is not assignable to type '{ [key: string]: Object; }'.
    The 'Object' type is assignable to very few other types. Did you mean to use the 'any' type instead?
      Index signature is missing in type 'Object'.
```

One option would be to create a separate types and use those, but we will need probably two of them since `.dockercfg` and `.dockerconfigjson` have different structure. Not sure if that's worth if we can use `any`

Reply:
Ah, OK. If it's an actual object and not the serialized string, then `any` is OK